01_fib/choohy/main_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestFibNeg11(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(-11)
+
+	expected := strconv.Quote(``)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Helper()
+		t.Errorf(expected)
+		t.Errorf(actual)
+	}
+}
+func TestFib0(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(0)
+
+	expected := strconv.Quote(``)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Helper()
```

Comment:
I'm not really sure what this `Helper()` method does, or if it is required. From reading the documentation:
> Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Helper may be called simultaneously from multiple goroutines.

Does that mean that this test is skipped? If this isn't adding anything to your tests, it might be best to remove it.